test/demo.html

```diff
@@ -116,6 +129,14 @@
     latexMath.latex(decodeURIComponent(location.hash.slice(1))).focus();
 });
 
+$(".insert-trigger").click(function () {
+  var latex = $(this).data('latex');
+  $(this).parent().find('.mathquill-math-field').each(function () {
+    var mathquill = MQ(this);
+    mathquill.write(latex);
+  });

```

Comment:
this appears to seek out every editable math field in the paragraph and write `\sqrt{4}` into it, including the editable math field in the button, is this not expected behavior?